easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -360,9 +366,9 @@ private static void runAnnotatedMethods(@NonNull Object object, int requestCode)
 
         while (clazz != null) {
             for (Method method : clazz.getDeclaredMethods()) {
-                if (method.isAnnotationPresent(AfterPermissionGranted.class)) {
```

Comment:
Intellij started complaining about this being a java 8 only feature. It shouldn't really matter since marshmallow came with 1.8, but I've made it backwards compatible anyway.